src/snoowrap.js

```diff
@@ -1462,15 +1462,16 @@ can get a post and a comment   *  @returns {Promise<Listing<Submission|Comment>>
     type = 'public',
     wiki_edit_age,
     wiki_edit_karma,
-    wikimode = 'modonly'
+    wikimode = 'modonly',
+    ...otherKeys
   }) {
     return this._post({
       uri: 'api/site_admin', form: {
         allow_images, allow_top, api_type, captcha, collapse_deleted_comments, comment_score_hide_mins, description,
         exclude_banned_modqueue, 'header-title': header_title, hide_ads, iden: captcha_iden, lang, link_type, name,
         over_18, public_description, public_traffic, show_media, show_media_preview, spam_comments, spam_links,
         spam_selfposts, spoilers_enabled, sr, submit_link_label, submit_text, submit_text_label, suggested_comment_sort,
-        title, type, wiki_edit_age, wiki_edit_karma, wikimode
+        title, type, wiki_edit_age, wiki_edit_karma, wikimode, ...otherKeys
```

Comment:
I'd put this on a new line for better readability
```suggestion
        title, type, wiki_edit_age, wiki_edit_karma, wikimode,
        ...otherKeys
```